aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/entities/Segment.java

```diff
@@ -43,6 +43,8 @@ static Segment noOp(TraceID traceId, AWSXRayRecorder recorder) {
     /**
      * @return the sampled
      */
+    @JsonIgnore
+    @Override
```

Comment:
Do we need to necessarily override the `isSampled()` method from `Entity`?
Is it possible for `Entity` types and subtypes to directly use this method from `Entity` without an override?